tools/packaging/kernel/build-kernel.sh

```diff
@@ -147,7 +147,7 @@ get_kernel() {
 	[ -n "${kernel_path}" ] || die "kernel_path not provided"
 	[ ! -d "${kernel_path}" ] || die "kernel_path already exist"
 
-	if [ "${conf_guest}" != "" ]; then
+	if [ "${conf_guest}" == "tdx" ]; then
```

Comment:
It seems like `get_tee_kernel` is supposed to be generic. Why don't we want to use it for SEV?